dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateNoEvidenceFoundAndIncomplete.kt

```diff
@@ -0,0 +1,63 @@
+package db.migration
+
+import db.migration.utils.DataTableEntity
+import db.migration.utils.getOrJavaNull
+import db.migration.utils.migrateCompanyAssociatedDataOfDatatype
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.json.JSONObject
+
+/**
+ * This migration script moves all SFDR Incomplete quality buckets to NoDataFound and removes the value NoEvidenceFound
+ * by putting NoDataFound into the quality bucket and setting the value to null
+ */
+class V19__MigrateNoEvidenceFoundAndIncomplete : BaseJavaMigration() {
+
+    /**
+     * Find all data points with a value of NoEvidenceFound and move it to the quality bucket or with quality Incomplete
+     * and move it to NoDataFound
```

Comment:
Ist aktuell etwas verwirrend.

Vorschlag:

Find all data points with a quality bucket of "NoEvidenceFound", set its value to null and its quality bucket to "NoDataFound".
Also find all data points with a quality bucket of "Incomplete" and set its quality bucket to "NoDataFound".

Reply:
angepasst